.github/workflows/ci-cd.yml

```diff
@@ -19,8 +19,6 @@ permissions:
 jobs:
     ci-cd:
         runs-on: ubuntu-latest
-        env:
-            TRIGGER_DEPLOY: ${{ startsWith(github.ref, 'refs/heads/master') || startsWith(github.ref, 'refs/heads/hotfix') || startsWith(github.ref, 'refs/heads/develop') || startsWith(github.ref, 'refs/heads/beta') }}
```

Comment:
`semantic-release` will check `release.config.js` to determine whether or not to deploy